### ISSUE TITLE
Log Files on Pantheon Doc Update 

### ISSUE THREAD

**Issue by dficker** [pantheon-systems/documentation#4945]
Re: [Log Files on Pantheon](https://pantheon.io/docs/logs/)

Priority: Low/Medium/High (choose one, remove the other options)

## Issue Description:

Looking at this part under FAQ: 
Why do I see requests coming from 10.x.x.x IPs in nginx-access.log?

The first IP address is no longer 10.x.x.x. Looks like it often starts with 35.x.x.x, but not always.

## Suggested Resolution

Maybe just reword it to "what's the first IP address from?" Or "Where's the IP of the visitor in the access log?" Most who have read logs before expect the first IP to be the IP of the visitor, but that's on a very simple routing system with no CDN in front of it.

**Comment by alexfornuto** [comment]
Will use [reserved IPs for documentation](https://tools.ietf.org/html/rfc5737) and add suggested rephrasing.

**Comment by alexfornuto** [comment]
Thanks @dficker!
